var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -15,12 +15,15 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     homepage = "https://www.gnu.org/software/emacs"
     git = "git://git.savannah.gnu.org/emacs.git"
     gnu_mirror_path = "emacs/emacs-24.5.tar.gz"
+    list_url = " https://ftpmirror.gnu.org/emacs/"
+    list_depth = 0
 
     maintainers("alecbcs")
 
     license("GPL-3.0-or-later")
```

Comment:
I did double-check this :-)

```suggestion
    license("GPL-3.0-or-later", checked_by="wdconinc")
```